trilium-note/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  server:
+    image: zadam/trilium:latest
```

Comment:
Here we'll need to pin the image to a particular tag along with its digest.

Reply:
Yup will do that here.